components/ai_chat/core/browser/conversation_handler_unittest.cc

```diff
@@ -1182,6 +1182,48 @@ TEST_F(ConversationHandlerUnitTest_NoAssociatedContent, GenerateQuestions) {
   testing::Mock::VerifyAndClearExpectations(engine);
 }
 
+TEST_F(ConversationHandlerUnitTest, SelectedLanguage) {
+  MockEngineConsumer* engine = static_cast<MockEngineConsumer*>(
+      conversation_handler_->GetEngineForTesting());
+
+  std::string expected_input1 = "Now stand aside, worthy adversary.";
+  std::string expected_input2 = "A scratch? Your arm's off!";
+  std::string expected_selected_language = "fr";
+
+  testing::Sequence s;
+  EXPECT_CALL(*engine,
+              GenerateAssistantResponse(false, StrEq(""), _, expected_input1,
+                                        StrEq(""), _, _))
+      .Times(1)
+      .InSequence(s)
+      .WillOnce(::testing::DoAll(
+          base::test::RunOnceCallback<5>(
+              mojom::ConversationEntryEvent::NewCompletionEvent(
+                  mojom::CompletionEvent::New("Tis but a scratch."))),
+          base::test::RunOnceCallback<5>(
+              mojom::ConversationEntryEvent::NewSelectedLanguageEvent(
+                  mojom::SelectedLanguageEvent::New(
+                      expected_selected_language))),
+          base::test::RunOnceCallback<6>(base::ok(""))));
+  EXPECT_CALL(*engine, GenerateAssistantResponse(
+                           false, StrEq(""), _, expected_input2,
+                           StrEq(expected_selected_language), _, _))
+      .InSequence(s)
```

Comment:
If there is only one sequence, we don't need to specify `InSequence`. `testing::Sequence s; in scope should be sufficient.

Reply:
Thanks. Removed the `InSequence` calls